src/sentry/incidents/endpoints/organization_incident_index.py

```diff
@@ -31,9 +31,17 @@ def get(self, request, organization):
         if envs:
             incidents = incidents.filter(alert_rule__snuba_query__environment__in=envs)
 
+        expand = request.GET.getlist("expand", [])
         query_alert_rule = request.GET.get("alertRule")
         if query_alert_rule is not None:
-            incidents = incidents.filter(alert_rule=query_alert_rule)
+            alert_rule_ids = [int(query_alert_rule)]
+            if "original_alert_rule" in expand:
```

Comment:
I'd consider adding an extra param here like `include_snapshots` or something along those lines - expand is mostly meant for adding extra data each thing we serialize, it isn't really intended for changing the actual result set

Reply:
Will do